### PR TITLE
Automatic commit: Move 'iot-edge-configure-account', 'iot-edge-enable…

### PR DESCRIPTION
**Pull request by marco-porru**
…', 'iot-edge-extension-helm', 'iot-edge-extension-install', 'iot-edge-extension-registry', 'iot-edge-get-gateway-details', 'iot-edge-install-gateway-persistence', 'iot-edge-install-smartsensing', 'iot-edge-onboard-node', 'iot-edge-onboard-thing' from QA to Production

### CODE REVIEW

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Details
+### You will learn
+  - How to enable IoT Edge applications in the SAP IoT tenant
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Enable IoT Edge Feature)]
+
+1.  In the navigation area **Tenant Administrator**, click on the **Tenant Configuration** application in the SAP Internet of Things Fiori Launchpad
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:28]: Better would be "Choose the **Tenant Configuration** application"

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    !![tenantconf](tenantconf.png)
+
+3.  Press **Save** to trigger the enablement. A screen informing you that enabling the feature you could incur in additional cost (accordingly with the IoT Edge pricing metrics).
+
+    !![confirm](confirm.png)
+
+4.  Tick the checkbox and press **Confirm** to enable the feature.
+
+5.  Wait the process complete. The process could take some minutes.
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:40]: "Wait for the process" or "Wait until the process is completed"

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    !![active](active.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Add Required User Roles)]
+
+Once the **Integrated Edge Functionality** is enabled, new roles will became available in your Business Technology Platform subaccount. These roles are required to make the IoT Edge applications visible in the SAP IoT Fiori Launchpad.
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:50]: "will become available"

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+1.  Open the sub-account containing your SAP IoT tenant in the Business Technology Platform Cockpit. In the left menu expand **Security** and click on **Role Collections**.
+
+    !![btpcollections](btpcollections.png)
+
+2.  Open the existing role collection for SAP IoT (i.e. `iot_role_collection`), or create a new one.
+
+    >Ensure the role collection is assigned to your BTP user. For further references see [Configure Role Collections for Users](https://help.sap.com/viewer/247022ddd1744053af376344471c0821/2109b/en-US)
+
+3.  Click on **Edit** and click on the textbox in the first line of **Role Name** column to start defining new roles.
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:60]: "and select the for line of the **Role Name** column"

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+    !![collection](collection.png)
+
+4.  Using the **Application Identifier** combo box, filter on the application identifier starting with `elmo-app-prod!`. Select the `EdgeLMAccess` role.
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:64]: "filter by application identifier"

File: tutorials/iot-edge-enable/iot-edge-enable.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+    !![elm](elm.png)
+
+
+4.  Using the **Application Identifier** combo box, filter on the application identifier starting with `les-admin-reuse-service!`. Select the `IoTEdge_Preprocessing` and `IoTEdge_Extensions` roles.
```
Comment (MichaelCzcz) [inline tutorials/iot-edge-enable/iot-edge-enable.md:69]: "filter by application identifier"

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for your pull request. Please make the requested changes.

**Comment by marco-porru** [comment]
Thanks for the comment @MichaelCzcz. I should have resolved all your comments.

**Comment by MichaelCzcz** [comment]
@marco-porru Please let @jmmargo know if you want the new tutorial to be part of a group/mission.

**Comment by marco-porru** [comment]
Thanks @MichaelCzcz

**Pull request merged**